schema/ext/vulnerability-1.0.xsd

```diff
@@ -0,0 +1,268 @@
+            </xs:element>
+        </xs:sequence>
+    </xs:complexType>
+
+    <xs:complexType name="vulnerability">
+        <xs:annotation>
+            <xs:documentation xml:lang="en">
+                Defines the structure of a vulnerability.
+            </xs:documentation>
+        </xs:annotation>
+        <xs:sequence>
+            <xs:element name="id" type="xs:normalizedString" minOccurs="1" maxOccurs="1">
+                <xs:annotation>
+                    <xs:documentation xml:lang="en">
+                        The id of the vulnerability as defined by the risk scoring methodology
+                        For example CVE-2019-15842 (of https://nvd.nist.gov/vuln/detail/CVE-2019-15842)
+                    </xs:documentation>
+                </xs:annotation>
+            </xs:element>
+            <xs:element name="source" minOccurs="0" maxOccurs="1">
+                <xs:complexType>
+                    <xs:annotation>
+                        <xs:documentation xml:lang="en">
+                            The source of the vulnerability where it is documented.
+                            Usually the name of the organization publishing vulnerability information
+                        </xs:documentation>
+                    </xs:annotation>
+                    <xs:sequence>
+                        <xs:element name="url" minOccurs="0" maxOccurs="1">
+                            <xs:annotation>
+                                <xs:documentation xml:lang="en">
+                                    The url of the vulnerability documentation as provided by the source
+                                    For example https://nvd.nist.gov/vuln/detail/CVE-2019-15842
+                                </xs:documentation>
+                            </xs:annotation>
+                            <xs:simpleType>
+                                <xs:restriction base="xs:string">
+                                    <xs:pattern value="(^(([^:/?#]+):)?(//([^/?#]*))?([^?#]*)(\?([^#]*))?(#(.*))?)"/>
+                                </xs:restriction>
+                            </xs:simpleType>
+                        </xs:element>
+                    </xs:sequence>
+                    <xs:attribute name="name" type="xs:normalizedString" use="required">
+                        <xs:annotation>
+                            <xs:documentation xml:lang="en">
+                                The name of the source. For example "National Vulnerability Database"
+                            </xs:documentation>
+                        </xs:annotation>
+                    </xs:attribute>
+                </xs:complexType>
+            </xs:element>
+            <xs:element name="vulnerabilityScore" minOccurs="0" maxOccurs="1">
+                <xs:annotation>
+                    <xs:documentation xml:lang="en">
+                        A vulnerability can have zero or more scores from different risk scoring methodologies.
```

Comment:
minor: `more scores` implies that there could be many of them, but `maxOccurs` is only one, might be confusing

Reply:
> minor: `more scores` implies that there could be many of them, but `maxOccurs` is only one, might be confusing

I've reworded it slightly differently 5c75f6f